frontend/src/concepts/pipelines/content/createRun/contentSections/RunTypeSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Alert, AlertActionCloseButton, Button, FormSection } from '@patternfly/react-core';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { PipelineRunTabTitle, PipelineRunType } from '~/pages/pipelines/global/runs';
+import {
+  CreateRunPageSections,
+  runPageSectionTitles,
+} from '~/concepts/pipelines/content/createRun/const';
+import { PipelineRunSearchParam } from '~/concepts/pipelines/content/types';
+
+interface RunTypeSectionProps {
+  runType: PipelineRunType;
+}
+
+export const RunTypeSection: React.FC<RunTypeSectionProps> = ({ runType }) => {
+  const navigate = useNavigate();
+  const { namespace } = useParams();
+  const [isAlertOpen, setIsAlertOpen] = React.useState(true);
+
+  let runTypeValue = 'Run once immediately after creation';
+  let alertProps = {
+    title: 'Go to Schedules to create schedules that execute recurring runs',
+    label: `Go to ${PipelineRunTabTitle.Schedules}`,
+    navSearch: `?${PipelineRunSearchParam.RunType}=${PipelineRunType.Scheduled}`,
+  };
+
+  if (runType === PipelineRunType.Scheduled) {
+    runTypeValue = 'Schedule recurring run';
+    alertProps = {
+      title: 'Go to Active runs to create a run that executes once immediately after creation.',
+      label: `Go to ${PipelineRunTabTitle.Active}`,
+      navSearch: `?${PipelineRunSearchParam.RunType}=${PipelineRunType.Active}`,
+    };
+  }
```

Comment:
Double check the alert title text, i think there is a mismatch from what the mocks have

Reply:
This was on purpose, since the mocks read "Go the Active runs", which is not grammatically correct. Other than that, I omitted the term "tab" since I don't think that's necessary to mention.